tests/integration_tests/test_mlflow.py

```diff
@@ -19,6 +24,16 @@ def _objective_func(trial: optuna.trial.Trial) -> float:
     return (x - 2) ** 2 + (y - 25) ** 2 + z
 
 
+def _multiobjective_func(trial: optuna.trial.Trial) -> Tuple[float, float]:
+
+    x = trial.suggest_uniform("x", low=-10, high=10)
+    y = trial.suggest_loguniform("y", low=1, high=10)
```

Comment:
A nit, but please use `suggest_float` instead of `suggest_uniform` and `suggest_loguniform`. We prefer `suggest_float` as can be seen in https://github.com/optuna/optuna/pull/2344.
```suggestion
    x = trial.suggest_float("x", low=-10, high=10)
    y = trial.suggest_float("y", low=1, high=10, log=True)
```

Reply:
Note: also TODO in `WeightsAndBiasesCallback`

Reply:
Addressed in 82b573ad6d32da113dcceb76fa3bee5fbd375325

Reply:
@toshihikoyanase I think we should mention such a preference at least in [coding style on Wiki](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions) or [`CONTRIBUTING.md`](https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md).

Reply:
Thank you for your update!

> we can mention such a preference at least in our coding style on Wiki or CONTRIBUTING.md.

@nzw0301 Sounds good! I guess this topic is too specific to be mentioned in [`CONTRIBUTING.md`](https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md), and I added [`Consistently use suggest_float` section](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#consistently-use-suggest_float) to the coding style wiki page. Please feel free to update it.

Reply:
@toshihikoyanase I agree with you. It looks perfect! Thanks!